Supplementary Docs (#7)
SixLabors/docs: articles/ImageSharp/GettingStarted.md

using (Image image = Image.Load("foo.jpg")) 
{
    // Resize the image in place and return it for chaining.
    // 'x' signifies the current image processing context.
    image.Mutate(x => x.Resize(image.Width / 2, image.Height / 2)); 

    // The library automatically picks an encoder based on the file extensions then encodes and write the data to disk.
    image.Save("bar.jpg"); 
} // Dispose - releasing memory into a memory pool ready for the next image you wish to process.
```

### How do I create a blank image for drawing on?
```c#
using SixLabors.ImageSharp;
using SixLabors.ImageSharp.PixelFormats;
using SixLabors.ImageSharp.Processing;

int width = 640;
int height = 480;

// Creates a new image with all the pixels set as transparent. 
using(var image = new Image<Rgba32>(width, height)) 
{
  // Do your drawing in here...

} // Dispose - releasing memory into a memory pool ready for the next image you wish to process.
```

### API Style
The easiest way to work with ImageSharp is to utilize our extension methods:
- @"SixLabors.ImageSharp" for basic stuff.
- @"SixLabors.ImageSharp.Processing" for `Mutate()` and `Clone()` 

> - All the processing extensions (eg. `Resize(...)`) are living within the @"SixLabors.ImageSharp.Processing" namespace. 

Reviewer: ```suggestion
 - All the processing extensions (eg. `Resize(...)`) live within the @"SixLabors.ImageSharp.Processing" namespace. 
```

> If you want to do image processing work, make sure you are `using` this namespace in your code.

Reviewer: ```suggestion
If you want to do image processing work, make sure you add `using SixLabors.ImageSharp.Processing;`.
```

### Why is [](xref:SixLabors.ImageSharp.Image`1?displayProperty=name) a generic class?

> An image is essentially a **generic 2D array of pixels** stored in a contiguous memory block. Check out the [Pixel Formats](PixelFormats.md) article for more details!

Reviewer: is the word 'generic' needed?
